   ml_ops/sm-datazone_import/cross-account-setup.md

```diff
@@ -0,0 +1,59 @@
+### Cross Account Setup
+In this scenario, we will use two accounts, a parent account hosting the DataZone domain, and a child account that contains
+a SageMaker Domain nd UserProfile that we would like to link, and import into the DataZone domain. Here AccountA is the parent
+account and AccountB is the child account.
+
+1. **[In the Parent Account]** Create a DataZone domain (make sure you are NOT using the Unified UI). It should say “Create a DataZone Domain”.
+2. **[In the Parent Account]** Create an association to the X-Account. 
+
+ - Click in to Domain - request Association by providing the the X-Account number.
+ - This will create a AWSRAMPermissionDataZoneDefault policy to allow access from the X-account. 
+ - In the X-Account, Accept the resource Share in the DataZone UI (Unified UI)
+
+3. **[In the Parent Account]** Create A Project, select the parent domain as the “DomainUnit”.
+4. **[In the Parent Account]** Add in the X-account user that will access this project.
+ 
+ - In the UserManagement tab, add the child user’s role (as an IAM user or SSO user) that requires access, from the X-account. Choose the AssociatedAccount option. 
+ - In the Projects tab, add the child user as a ProjectMember. They should be available in the DropDown menu. Set the respective permissions. 
+
+> NOTE :: If this step (a.) is not done, the X-account user will see this error in their projects tab when clicking into the Assocated Domain  
+
+```
+Not a DataZone user
+You cannot view or create a project because you have not been added 
+as a Amazon DataZone user. Please contact your domain admin to add 
+your IAM role: arn:aws:iam::211125770549:role/Admin as a DataZone user.
+```
```

Comment:
"If this step (a.)" - you mean step "(1.)"? looking at markdown above i'm not seeing step "a"
"this error" -> "the following error"

Reply:
Fixed.